docs/source/llms/transformers/task.rst

```diff
@@ -0,0 +1,208 @@
+    # >> text-generation
+
+    # Model signature is set to the chat API spec
+    print(model_info.signature)
+    # >> inputs:
+    # >>   ['messages': Array({content: string (required), name: string (optional), role: string (required)}) (required), 'temperature': double (optional), 'max_tokens': long (optional), 'stop': Array(string) (optional), 'n': long (optional), 'stream': boolean (optional)]
+    # >> outputs:
+    # >>   ['id': string (required), 'object': string (required), 'created': long (required), 'model': string (required), 'choices': Array({finish_reason: string (required), index: long (required), message: {content: string (required), name: string (optional), role: string (required)} (required)}) (required), 'usage': {completion_tokens: long (required), prompt_tokens: long (required), total_tokens: long (required)} (required)]
+    # >> params:
+    # >>     None
+
+    # The model can be served with the OpenAI-compatible inference API
+    pyfunc_model = mlflow.pyfunc.load_model(model_info.model_uri)
+    prediction = pyfunc_model.predict(
+        {
+            "messages": [
+                {"role": "system", "content": "You are a bot."},
+                {"role": "user", "content": "Hello, how are you?"},
+            ],
+            "temperature": 0.5,
+            "max_tokens": 200,
+        }
+    )
+    print(prediction)
+    # >> [{'choices': [{'finish_reason': 'stop',
+    # >>               'index': 0,
+    # >>               'message': {'content': 'I'm doing well, thank you for asking.', 'role': 'assistant'}},
+    # >>   'created': 1719875820,
+    # >>   'id': '355c4e9e-040b-46b0-bf22-00e93486100c',
+    # >>   'model': 'gpt2',
+    # >>   'object': 'chat.completion',
+    # >>   'usage': {'completion_tokens': 7, 'prompt_tokens': 13, 'total_tokens': 20}}]
+
+Note that the input and output modifications only apply when the model is loaded with :py:func:`mlflow.pyfunc.load_model()` (e.g. when 
+serving the model with the ``mlflow models serve`` CLI tool). If you want to load just the raw pipeline, you can
+always do so via :py:func:`mlflow.transformers.load_model()`.
+
+Provisioned Throughput on Databricks Model Serving
+--------------------------------------------------
+
+`Provisioned Throughput <https://docs.databricks.com/en/machine-learning/foundation-models/deploy-prov-throughput-foundation-model-apis.html>`_ on Databricks Model Serving is a capability that optimize inference performance for foundation models with performance guarantees. To serve Transformers models with provisioned throughput, specify ``llm/v1/xxx`` task type when logging the model. MLflow logs the required metadata to enable provisioned throughput on Databricks Model Serving.
+
+
+.. tip::
+
+    When logging foundation models, you can use ``save_pretrained=False`` to avoid creating a local copy of the model weights for saving time and disk space. Please refer to the :ref:`documentation <transformers-save-pretrained-guide>` for more details.
```

Comment:
```suggestion
    When logging large models, you can use ``save_pretrained=False`` to avoid creating a local copy of the model weights for saving time and disk space. Please refer to the :ref:`documentation <transformers-save-pretrained-guide>` for more details.
```

`large models` makes more sense because `foundational` is not necessarily `large` (they are usually large tho)?

Reply:
makes sense!